src/lib/battery/battery.cpp

```diff
@@ -194,6 +194,10 @@ void Battery::sumDischarged(const hrt_abstime &timestamp, float current_a)
 
 void Battery::estimateStateOfCharge(const float voltage_v, const float current_a)
 {
+	if (_params.n_cells == 0) {
```

Comment:
Set `_state_of_charge` and `_state_of_charge_volt_based` to 0?

Reply:
Yes, but I'll set it to -1.0f instead.